### ISSUE TITLE
Create read side HTTP API endpoints for scholarly content resources (categories, journals)

### ISSUE THREAD

**Issue by GiancarloFusiello** [libero/publisher#236]
Prerequisite:  https://github.com/libero/publisher/issues/235

**Tasks**
- [x] Create RESTful HTTP API endpoints for each model allowing ~CRUD~ read operations using HTTP methods and data in JSON format

**Comment by giorgiosironi** [comment]
After review of PR, limited the scope to just the read side; will define a separate ticket for the write side after understanding the interaction with `jats-ingester`.

**Comment by giorgiosironi** [comment]
Can merge the current PR and turn off the APIs created by the framework separately.

**Comment by giorgiosironi** [comment]
Still need to turn off admin and auth in a separate PR.

**Comment by giorgiosironi** [comment]
Splitting https://github.com/libero/publisher/issues/241
